definitions/divorce/json/SearchInputFields.json

```diff
@@ -118,6 +118,20 @@
     "Label": "Supplementary evidence handled",
     "DisplayOrder": 17
   },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
can we use today's date which can help readers to know roughly when this field is introduced ? the same applicable to other `LiveFrom` fields

Reply:
sure, I'll update now